src/bitboard/mod.rs

```diff
@@ -12,9 +12,9 @@ use std::{
 	mem::size_of,
 	ops::Mul,
 	ops::{
-		Add, BitAnd, BitAndAssign, BitOr, BitOrAssign, BitXor, BitXorAssign, Index, Not, Shl, Shr,
-		Sub,
-	},
+		Add, BitAnd, BitAndAssign, BitOr, BitOrAssign,
+		BitXor, BitXorAssign, Index, Not, Shl, Shr, Sub
+	}
```

Comment:
Still not sexy but should do the job (meaning you can keep it like this even If I don't like it)